src/main/java/reposense/configparser/DateParser.java

```diff
@@ -0,0 +1,33 @@
+package reposense.configparser;
+
+import java.util.Date;
+
+import reposense.exceptions.ParseException;
+import reposense.util.Constants;
+
+/**
+ * Verifies and parses a string-formatted date to a Date object.
+ */
+public class DateParser extends Parser<Date, String> {
+    private static final String PARSE_EXCEPTION_MESSAGE_INVALID_DATE = "Invalid Date";
+    private static final String PARSE_EXCEPTION_MESSAGE_INVALID_DATE_STRING_FORMAT =
```

Comment:
Can just mix this 2 together, since the `PARSE_EXCEPTION_MESSAGE_INVALID_DATE` is used only in this String formatting.